examples/src/main/java/com/traveltime/examples/TimeFilterProtoExample.java

```diff
@@ -0,0 +1,61 @@
+package com.traveltime.examples;
+
+import com.traveltime.sdk.TravelTimeSDK;
+import com.traveltime.sdk.auth.TravelTimeCredentials;
+import com.traveltime.sdk.dto.common.Coordinates;
+import com.traveltime.sdk.dto.requests.TimeFilterFastProtoRequest;
+import com.traveltime.sdk.dto.requests.proto.*;
+import javafx.util.Pair;
+import lombok.val;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static io.vavr.API.*;
+import static io.vavr.Patterns.$Left;
+import static io.vavr.Patterns.$Right;
+
+/**
+ * Example how to get 3 closest shops using proto
```

Comment:
Change to:
Example showing how to find 3 closest shops by travel time given a list of shops with locations. List is generated randomly. Here we are using `walking` transportation mode, but you can use different ways of transportation, for example: public transport or driving. This example is similar to `TimeFilterExample` but is using protobuf to achieve much faster response time. If you're looking for real-time search we recommend to use protobuf. You can find TravelTime protobuf benchmarks at - https://github.com/traveltime-dev/traveltime-benchmarks